src/ML/pytorch/LinRegMain/linRegMain.cc

```diff
@@ -33,6 +33,6 @@ int main() {
 	//Start of linear Regression learning with LearnedPolicy by setting agent's policy to lpol
     //agent.setPolicy(&lpol);
     
-    TrainingSet TS(3, 100, 20, 64, 0.1);
+    TrainingSet TS(10, 100, 20, 64, 0.1);
     TS.train(&agent, lpol);
```

Comment:
You still use the `rnd` flag when creating the agent. Better directly start with a `LearningPolicy` and also don't create it outside. Use the agents' enum.

Reply:
We have tried but met with some difficulty so we focused on this issue first. 
We might make a seperate task for this so it doesn't hinder our progress.